File: tang-commons/src/main/java/com/tang/commons/constants/CachePrefix.java
package com.tang.commons.constants;

/**
 * 缓存前缀
 *
 * @author Tang
 */
public class CachePrefix {

    private CachePrefix() {
    }

    /**
     * 缓存前缀
     */
    public static final String SYSTEM = "tang-boot:";

    /**
     * 登陆用户信息缓存前缀
     */
    public static final String LOGIN_TOKENS = SYSTEM + "login_tokens:";

    /**
     * 登陆用户ID缓存前缀
     */
    public static final String LOGIN_USER_ID = SYSTEM + "login_user_id:";

    /**
     * 字典数据缓存前缀
     */
    public static final String DICT_TYPE = SYSTEM + "dict_type:";

    /**
     * 验证码缓存前缀
     */
    public static final String CAPTCHA = SYSTEM + "captcha:";

}

File: tang-commons/src/main/kotlin/com/tang/commons/constants/CachePrefix.kt
package com.tang.commons.constants

/**
 * 缓存前缀
 *
 * @author Tang
 */
object CachePrefix {

    /**
     * 缓存前缀
     */
    const val SYSTEM = "tang-boot:"

    /**
     * 登陆用户信息缓存前缀
     */
    const val LOGIN_TOKENS = SYSTEM + "login_tokens:"

    /**
     * 登陆用户ID缓存前缀
     */
    const val LOGIN_USER_ID = SYSTEM + "login_user_id:"

    /**
     * 字典数据缓存前缀
     */
    const val DICT_TYPE = SYSTEM + "dict_type:"

    /**
     * 验证码缓存前缀
     */
    const val CAPTCHA = SYSTEM + "captcha:"

}
